lib/fuzzer.zig

```diff
@@ -460,64 +124,46 @@ export fn fuzzer_init(cache_dir_struct: Fuzzer.Slice) void {
 
     const pcs = pcs_start[0 .. pcs_end - pcs_start];
 
-    const cache_dir_path = cache_dir_struct.toZig();
-    const cache_dir = if (cache_dir_path.len == 0)
-        std.fs.cwd()
-    else
-        std.fs.cwd().makeOpenPath(cache_dir_path, .{ .iterate = true }) catch |err| {
-            fatal("unable to open fuzz directory '{s}': {s}", .{ cache_dir_path, @errorName(err) });
-        };
+    fuzzer = Fuzzer.init(cache_dir, pc_counters, pcs);
```

Comment:
While decl literals are nice and I would write it like that, this is ultimately a relatively unimportant stylistic choice. As such, OP, feel free to disregard this comment.

Reply:
I asked the user not to make pull request reviews 10 minutes before this comment. Since they disregarded my request, I will now make the request again via the moderation tools.